index.html

```diff
@@ -13,7 +21,7 @@
             margin: 0;
             padding: 0;
             height: 100%;
-        }
+        }z
```

Comment:
This feels like a typo? (or is there some obscure css i'm missing here?)

Reply:
Yep this is a typo. Thanks for pointing it out